interfaces/IStaking.sol

```diff
@@ -0,0 +1,103 @@
+//SPDX-License-Identifier: Unlicense
+
+pragma solidity 0.8.7;
+
+interface IStaking {
+
+    /**
+     * @dev Store info about beneficiaries
+     *
+     * @param startDate - when user call 'stake' in first time
+     * @param finishDate - when user call 'unstake' in lastTime
+     * @param stakingAmount - how many tokens user staked
+     * @param userRewardPerTokens - how many reward user accumulated
+     * @param userRewardPerTokens - how many reward user collect
+     */
+    struct Investor {
+        uint256 startDate;
```

Comment:
startDate and finishDate should be general for all investors, not specific for each